components/linkedin/actions/create-text-post-user/create-text-post-user.mjs

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+  key: "linkedin_create-text-post-user",
+  name: "Create a Text Post (User)",
+  description: "Create shares on LinkedIn using text, URLs, and images. [See the docs](https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/share-on-linkedin#creating-a-share-on-linkedin) for more information",
+  version: "0.0.11",
+  type: "action",
+  props: {
```

Comment:
Prop definitions should typically live in the app file. See https://pipedream.com/docs/components/guidelines/#app-files

Reply:
done